Header header
int32 index
int32 value_type
int32 value
